### PR TITLE
feat: ui kit

### PR DESCRIPTION
**Pull request by alexzavgorodnii**
localhost_3003_
localhost_3003_ (1)

### CODE REVIEW

File: src/shared/components/Button/styles.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  height: 54px;
+  font-size: 16px;
+  line-height: 20px;
+  padding: 0 30px;
+`;
+
+const primaryBaseStyle = `
+  border: 2px solid ${Colors.ORANGE_DARK};
+  background-color: ${Colors.ORANGE_DARK};
+  color: ${Colors.WHITE};
```
Comment (olegdolenchuk) [inline src/shared/components/Button/styles.ts:25]: I think you need to add minimal animation for all type buttons

File: src/shared/Typography/styles.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import Colors from '../colors';
+
+export const H1 = styled.h1`
+  font-family: 'Montserrat', sans-serif;
```
Comment (olegdolenchuk) [inline src/shared/Typography/styles.ts:5]: I think you can create a variable for the font so as not to duplicate it

File: src/shared/components/Button/styles.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+import Colors from '../../colors';
```
Comment (oknelvapi) [inline src/shared/components/Button/styles.ts:2]: You could use absolute paths: 
`import Colors from 'shared/colors';`

File: src/shared/Typography/styles.ts (hunk 2 above)
Comment (alexzavgorodnii) [reply src/shared/Typography/styles.ts:5]: done

File: src/shared/components/Button/styles.ts (hunk 3 above)
Comment (alexzavgorodnii) [reply src/shared/components/Button/styles.ts:2]: done

File: src/shared/components/Button/styles.ts (hunk 1 above)
Comment (alexzavgorodnii) [reply src/shared/components/Button/styles.ts:25]: done

File: src/shared/components/Button/styles.ts (hunk 3 above)
Comment (alexzavgorodnii) [reply src/shared/components/Button/styles.ts]: done

File: src/shared/components/Button/component.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import { ButtonWrapper, ButtonVariant } from './styles';
+
+type ButtonProps = {
+  variant: ButtonVariant;
+  title: string;
+};
+
+const Button: React.FC<ButtonProps> = ({ variant, title }: ButtonProps) => {
+  return <ButtonWrapper variant={variant}>{title}</ButtonWrapper>;
+};
```
Comment (oknelvapi) [inline src/shared/components/Button/component.tsx:11]: Btw, if you don't have any side effects, you can write without `return`:
```javascript
const Button: React.FC<ButtonProps> = ({ variant, title }: ButtonProps) => ( <ButtonWrapper variant={variant}>{title}</ButtonWrapper>;
);
`

Comment (alexzavgorodnii) [reply src/shared/components/Button/component.tsx]: fixed

**Pull request merged**